Header header
uint16  ctrlWord
uint16  maxTorque
int32  targetPos
int32  targetVel

uint16  coeStatus
int32  actualPos
int32  actualVel
